src/main/java/uk/nhs/digital/nhsconnect/nhais/translator/amendment/mappers/AmendmentToEdifactMapper.java

```diff
@@ -38,6 +38,7 @@ protected void validateNonEmptyValues(List<Optional<AmendmentPatch>> amendmentPa
         var invalidAmendmentPaths = amendmentPatches.stream()
             .flatMap(Optional::stream)
             .filter(AmendmentToEdifactMapper::amendmentPatchRequiringValue)
+            .filter(amendmentPatch -> amendmentPatch.getValue() != null)
```

Comment:
How could it be null?

Reply:
When value in JSON is set explicitly to null
````
{
(...)
	{
      "op": "replace",
      "path": "/address/0/line/2",
      "value": null
    }
(...)
}
````
As Alicja told me yesterday it is valid request.